action.yml

```diff
@@ -6,7 +6,7 @@ inputs:
   docker-registry:
     description: 'Docker Registry'
     required: true
-    default: 'registry.staffbase.com'
+    default: 'staffbase.jfrog.io'
```

Comment:
i would rather change that here: https://github.com/Staffbase/gha-workflows/blob/main/.github/workflows/template_gitops.yml

Reply:
I'm not sure why this is public at all, but the goal is to reduce the configuration for developers, so it would be counterproductive to not set a default.